README.md

```diff
@@ -31,7 +31,6 @@ pip install proselint
 - [x] [Sublime Text](https://github.com/amperser/proselint/tree/master/plugins/sublime/SublimeLinter-contrib-proselint)
 - [x] [Atom Editor](https://github.com/smockle/linter-proselint) (thanks to [Clay Miller](https://github.com/smockle)).
 - [x] [Emacs via Flycheck](https://github.com/amperser/proselint/tree/master/plugins/flycheck) (thanks to [Aaron Jacobs](https://github.com/atheriel))
-- [x] [Vim](https://github.com/amperser/proselint/tree/master/plugins/vim) (thanks to [Matthias Bussonnier](https://github.com/Carreau))
```

Comment:
Yes, agreed since vim-syntastic supports proselint across a variety of configurations
Suggested change to:

 - [x] [Vim](https://github.com/vim-syntastic/syntastic) (Thanks to @lcd047 & @Carreau)

?